app/controllers/digital-wallet/post-disable-google-pay-controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+// Local dependencies
+const paths = require('../../paths')
+const { ConnectorClient } = require('../../services/clients/connector_client')
+const auth = require('../../services/auth_service')
+const { CORRELATION_HEADER } = require('../../utils/correlation_header.js')
+const connector = new ConnectorClient(process.env.CONNECTOR_URL)
+
+module.exports = (req, res) => {
+  const gatewayAccountId = auth.getCurrentGatewayAccountId(req)
+  const correlationId = req.headers[CORRELATION_HEADER] || ''
+  const enableGooglePayBoolean = connector.toggleGooglePay(gatewayAccountId, false, correlationId)
```

Comment:
This is confusing to read, if we are not planning to do anything with the value that is eventually propagated there should be no reason to assign this

```suggestion
connector.toggleGooglePay(gatewayAccountId, false, correlationId)
  .then(() => { 
```